src/windows/win-crowdstrike-fix-bootloop.ps1

```diff
@@ -0,0 +1,25 @@
+. .\src\windows\common\setup\init.ps1
+. .\src\windows\common\helpers\Get-Disk-Partitions.ps1
+
+$partitionlist = Get-Disk-Partitions
+$actionTaken = $false
+
+forEach ( $partition in $partitionlist )
+{
+    $driveLetter = ($partition.DriveLetter + ":")
+    $corruptFiles = "$driveLetter\Windows\System32\drivers\CrowdStrike\C-00000291*.sys"
```

Comment:
I checked the statement on https://www.crowdstrike.com/blog/statement-on-falcon-content-update-for-windows-hosts/ 

It says:
`Locate the file matching “C-00000291*.sys”, and delete it.` 

doesn't the asterisk mean a regex? From what I know Windows doesn't allow `*` in its file names right?

Reply:
Powershell can glob file paths https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_wildcards?view=powershell-7.4#long-description

You can test the behavior locally with
```powershell
# First test is False (no files found)
Test-Path -Path ".\test*.txt"

echo "test1" > test1.txt
echo "test2" > test2.txt

# Second test is True (files matched)
Test-Path -Path ".\test*.txt"

Remove-Item -Path ".\test*.txt"

# Third test is False (files have been deleted)
Test-Path -Path ".\test*.txt"
```

Reply:
```suggestion
    $corruptFiles = Get-ChildItem -Path "$driveLetter\Windows\System32\drivers\CrowdStrike\" -Filter "C-00000291*.sys"
```
you'll need to test this

Reply:
Cool, thanks for clarifying and I don't have a windows machine in reach at this moment

Reply:
I reached out to the team owning this so they should be on this soon